tests/ignite/distributed/utils/__init__.py

```diff
@@ -228,6 +228,38 @@ def _test_distrib_barrier(device):
     assert tt.item() == true_res + 10.0
 
 
+def _test_distrib_new_group(device):
+
+    if idist.get_world_size() > 1 and idist.utils.backend() is not None:
+        bnd = idist.backend()
+        ranks = [0, 1]
+        if idist.has_native_dist_support and bnd in ("nccl", "gloo", "mpi"):
+
+            assert idist.new_group(ranks).rank == dist.new_group(ranks=ranks).rank
+        elif idist.has_xla_support and bnd in ("xla-tpu"):
+
+            assert idist.new_group(ranks) == [ranks]
+        elif idist.has_hvd_support and bnd in ("horovod"):
+            from horovod.common.process_sets import ProcessSet
+
+            assert idist.new_group(ranks).rank == ProcessSet(ranks).rank
+
+    else:
```

Comment:
Instead of else I think you want to do `elif idist.utils.backend() is None` ?

Reply:
I tried to use `else` as we discussed.